libcudacxx/include/cuda/__memory_resource/resource_ref.h

```diff
@@ -347,43 +365,59 @@ class basic_resource_ref
       , __vtable(__ref)
   {}
 
+  //! @brief Equality comparison between two \c basic_resource_ref
+  //! @param __rhs The other \c basic_resource_ref
+  //!
+  //! @return Checks whether both resources have the same equality function stored in their vtable and if so returns
+  //! that equality comparison. Otherwise returns false.
```

Comment:
Suggestion
```suggestion
  //! @return Checks whether both resources have the same equality function stored in their vtable and if so returns
  //! the result of that equality comparison. Otherwise returns false.
```